lazer/sdk/js-solana/package.json

```diff
@@ -0,0 +1,63 @@
+{
+  "name": "@pythnetwork/pyth-lazer-solana-sdk",
+  "version": "0.1.0",
+  "description": "Pyth Lazer Solana SDK",
+  "publishConfig": {
+    "access": "public"
+  },
+  "files": [
+    "dist/**/*"
+  ],
+  "main": "./dist/cjs/index.js",
+  "types": "./dist/cjs/index.d.ts",
+  "exports": {
+    "import": {
+      "types": "./dist/esm/index.d.ts",
+      "default": "./dist/esm/index.js"
+    },
+    "require": {
+      "types": "./dist/cjs/index.d.ts",
+      "default": "./dist/cjs/index.js"
+    }
+  },
+  "scripts": {
+    "build:cjs": "swc src -d dist/cjs --strip-leading-paths --copy-files -C module.type=commonjs && echo '{\"type\":\"commonjs\"}' > dist/cjs/package.json",
+    "build:esm": "swc src -d dist/esm --strip-leading-paths --copy-files -C jsc.experimental.keepImportAttributes=true && echo '{\"type\":\"module\"}' > dist/esm/package.json",
+    "fix:lint": "eslint --fix . --max-warnings 0",
+    "test:lint": "eslint . --max-warnings 0",
+    "test:types": "tsc",
+    "test:format": "prettier --check .",
+    "fix:format": "prettier --write .",
+    "doc": "typedoc --out docs/typedoc src"
+  },
+  "devDependencies": {
+    "@cprussin/eslint-config": "catalog:",
+    "@cprussin/tsconfig": "catalog:",
+    "@swc/cli": "^0.7.8",
+    "@swc/core": "^1.13.2",
+    "@types/node": "^18.19.54",
+    "eslint": "catalog:",
+    "prettier": "catalog:",
+    "typedoc": "^0.26.8",
```

Comment:
I suggest moving these dependencies up to `pnpm-workspace.yaml` and marking them as `catalog:` dependencies; in general I think we should only be using numbered dependencies for places where some package needs to be held back from the rest of the monorepo for some reason.